packages/types/dbinfo.d.ts

```diff
@@ -118,9 +118,24 @@ export interface ViewInfo extends SqlObjectInfo {
   columns: ColumnInfo[];
 }
 
-export interface ProcedureInfo extends SqlObjectInfo {}
+export interface ParameterInfo {
```

Comment:
Please make it similarly as ColumnInfo

add: extends NamedObjectInfo
this holds information about parent entity (procedure)

remove parentObjectId, routineName, pureName
parentObjectId could be used for pairing internally, but it should be not part of typings, it is only internal attribute for plugin-mssql

add parameterName: string - name of parameter

remove maxLength, precision, scale - this should be already stored in dataType
remove fullDataType, only dataType should be stored

There is definition of precision, scale, length in ColumnInfo, this is a bit bug, it should be never used nor filled, only dataType and displayedDataType is used